native_client/javascript/client.ts

```diff
@@ -1,48 +1,40 @@
 #!/usr/bin/env node
-'use strict';
-
-const Fs = require('fs');
-const Sox = require('sox-stream');
-const Ds = require('./index.js');
-const argparse = require('argparse');
-const MemoryStream = require('memory-stream');
-const Wav = require('node-wav');
-const Duplex = require('stream').Duplex;
-const util = require('util');
-
-var VersionAction = function VersionAction(options) {
-  options = options || {};
-  options.nargs = 0;
-  argparse.Action.call(this, options);
-}
-util.inherits(VersionAction, argparse.Action);
 
-VersionAction.prototype.call = function(parser) {
-  console.log('DeepSpeech ' + Ds.Version());
-  let runtime = 'Node';
-  if (process.versions.electron) {
-    runtime = 'Electron';
+import Ds from "./index";
+import * as Fs from "fs";
+import Sox from "sox-stream";
+import * as argparse from "argparse";
+
+const MemoryStream = require("memory-stream");
+const Wav = require("node-wav");
+const Duplex = require("stream").Duplex;
+
+class VersionAction extends argparse.Action {
+  call(parser: argparse.ArgumentParser, namespace: argparse.Namespace, values: string | string[], optionString: string | null) {
+    console.log('DeepSpeech ' + Ds.Version());
+    let runtime = 'Node';
+    // @ts-ignore: TS2339: Property 'electron' does not exist on type 'ProcessVersions'.
```

Comment:
I could not find  a better solution yet, maybe we need some types from electron?

Reply:
Looks like you do need to add Electron ([SA Answer](https://stackoverflow.com/questions/47398555/electron-typescript-compilation)).

Reply:
Thanks!

Reply:
It works, just required some `/// reference` to bve added after installing the module dep.